cmd/notation/sign.go

```diff
@@ -99,14 +109,28 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 		return err
 	}
 	ociImageManifest := cmdOpts.signatureManifest == signatureManifestImage
+	if cmdOpts.localContent {
+		return signLocal(ctx, cmdOpts, signer, ociImageManifest)
+	}
+	return signRemote(ctx, cmdOpts, signer, ociImageManifest)
```

Comment:
`signLocal` and `signRemote` should be common and should only have slight difference. Thus refactoring is requested.